config/templates/Vagrantfile

```diff
@@ -43,4 +43,11 @@ Vagrant.configure(2) do |config|
         #vb.memory = "8192"
         #vb.cpus = "4"
     end
+
+    case File.basename(Dir.getwd)
+    when "templates"
```

Comment:
Working config is in /userpatches. Only that one should be selected. If it is not existing it has to be copy there. Do we need to check "templates" dir?

Reply:
That's for Windows users. They can't execute `./compile.sh vagrant`.